src/v3.ts

```diff
@@ -166,19 +166,39 @@ export default function generateTypesV3(
           output += `parameters: {\n`;
           const allParameters: Record<
             string,
-            Record<string, OpenAPI3Parameter>
+            Record<string, OpenAPI3Parameter | string>
           > = {};
           responses.parameters.forEach((p) => {
-            if (!allParameters[p.in]) allParameters[p.in] = {};
-            // TODO: handle $ref parameters
-            if (p.name) {
-              allParameters[p.in][p.name] = p;
+            if ("$ref" in p) {
+              // return;
```

Comment:
Remove?

Reply:
done in https://github.com/drwpow/openapi-typescript/pull/340